pure.zsh

```diff
@@ -582,6 +583,29 @@ prompt_pure_setup() {
 	# to be available, it was added in Zsh 5.3.
 	autoload -Uz +X add-zle-hook-widget 2>/dev/null
 
+	# most used colors
```

Comment:
We should add `zmodload zsh/zutil` a few lines above to make sure `zstyle` is loaded when we try to use it.